src/unistd.rs

```diff
@@ -819,6 +821,619 @@ pub fn mkstemp<P: ?Sized + NixPath>(template: &P) -> Result<(RawFd, PathBuf)> {
     Ok((fd, PathBuf::from(pathname)))
 }
 
+/// Variable names for `pathconf`
+///
+/// Note: POSIX 1003.1-2008 standardizes all of these variables, but some OSes
+/// choose not to implement variables that cannot change at runtime.
+///
+/// # References
+///
+/// http://pubs.opengroup.org/onlinepubs/9699919799/functions/pathconf.html
```

Comment:
Please make these bulleted, as they just show up on a single line when docs are generated. Also if you could make them links, use the title of the manpage as the title for them or providing a summary would be great.